ccp/7_deploy_osh/roles/suse-osh-deploy/tasks/ceph-secrets.yml

```diff
@@ -0,0 +1,71 @@
+---
+- name: Create ceph secrets
+  copy:
+    dest: "/tmp/ceph-secrets.yaml"
+    content: |
```

Comment:
Content provided here seems like a good candidate to be provided as template file. This yaml content can be easily edited and verified in a separate file. 
Is there specific reason to have them as content?

Reply:
It was written as a series of tasks initially and therefore the inline was easier. Now that it's written as a template, we should move to template:

Good catch.